docs/docs/manual/columnediting.md

```diff
@@ -0,0 +1,128 @@
+
+## Add column based on this column
+
+This selection will open up an [expressions](expressions) window where you can transform the data from this column (using `value`) or write a complex expression that takes information from any number of columns or from external reconciliation sources. 
+
+One common use of this function is to join information from two projects together in one place. Just like referring from one table to another table in an entity-relationship database, you need a key that exists in both. 
+
+The expression is
+
+```cell.cross('arg1','arg2').cells['arg3'].value[arg4]```
+
+where 
+
+*   **arg1** is the name of the project you want to pull data from (you will get an error in the Preview window if there are multiple projects with the identified name)
+*   **arg2** is the column in that project with matching values to the column in the current project
+*   **arg3** is the column in that project you’d like to copy over (you can only specify one column at a time)
+*   and **arg4** is which value in that column to import (most likely 0).
+
+Learn more about [cross()](expressions#cross) and other GREL functions to use in this window on the [Expressions](expressions) page. 
+
+Some of the other most common ways to add a new column based on an existing one are separate functions, and are explained below.
+
+
+## Add column by fetching URLs
+
+Through the "Add column by fetching URLs" function, OpenRefine supports the ability to fetch HTML or data from web pages or services. In this operation you will be taking strings from your selected column and inserting them into URL strings. This presumes your chosen column contains parts of paths to valid HTML pages or files online. 
+
+If you have a column of URLs and watch to fetch the information that they point to, you can simply run the expression as `value`. If your column has, for example, unique identifiers for Wikidata entities (numerical values starting with Q), you can download the JSON-formatted metadata about each entity with
+
+```“https://www.wikidata.org/wiki/Special:EntityData/” + value + “.json”```
+
+or whatever metadata format you prefer. [Information about these Wikidata options can be found here](https://www.wikidata.org/wiki/Wikidata:Data_access).
+
+This service is more useful when getting metadata files instead of HTML files, but you may wish to work with a page’s entire HTML contents and then parse out information from that. Be cautioned that the fetching process can take quite some time and that servers may not want to fulfill hundreds or thousands of page requests in seconds. 
+
+Fetching allows you to set a “throttle delay” which determines the amount of time between requests. The default is 5 seconds per row in your dataset (5000 milliseconds), so you can estimate how long it will take to work through the entire column using that number. We recommend leaving this value at 5000 or greater. 
```

Comment:
5 seconds is very long for most web services. They should certainly be polite, but reducing this to 1 second or less if fine for the rate limit on most web services.

Reply:
Is "leaving this value at 1000 or greater" okay? Or should we provide some guidance on how to calculate this?

Would it also be worth changing the default in the program?

Reply:
In our meeting today we agreed, so there's an issue for this now: https://github.com/OpenRefine/OpenRefine/issues/3127

In terms of guidance we'll simply cut that "recommend" sentence.